config/deploy/staging.hcl

```diff
@@ -61,9 +66,18 @@ job "dpulc-staging" {
       driver = "podman"
       config {
         image = "ghcr.io/pulibrary/dpul-collections:${ var.branch_or_sha }"
-        ports = ["http"]
+        ports = ["http", "epmd"]
         force_pull = true
       }
+      resources {
+        cpu    = 2000
+        memory = 1000
```

Comment:
how did we chose these numbers? what is the unit? can we document a way to assess these numbers moving forward?

Reply:
While indexing in Nomad CPU would never break 2ghz, memory wouldn't break 1 GB, so I adjusted them here. Nomad was killing the indexer for going above the previous defaults (which are like 500 mhz/300mb)

Reply:
Maybe an ADR about this??

Reply:
This got in main as part of the indexer, which has its own ADR. Good?